formbuilder/components/builder/Form.js

```diff
@@ -29,7 +29,7 @@ export default function EditableForm(props) {
   let saveButtonValue = "Create form";
   if (props.status == "pending") {
     saveButtonValue = <div>Create form <i className="spin glyphicon glyphicon-refresh" /></div>;
-  } else if (props.status == "done"){
+  } else if (props.status == "done") {
```

Comment:
I actually tried but I'm not sure it's valuable :the error is already shown to the user and the button is here to do an action, having a cross in it is counter intuitive